### DISCUSSION TITLE
Mica Backdrop

### DISCUSSION THREAD

**Discussion by sapphired18** [File-New-Project/EarTrumpet#1014] [Ideas]
using acrylic in windows 11 takes more energy (resources) than using mica, I got lag when Eartrumpet overlay open

**Comment by riverar** [comment]
All Windows Taskbar flyouts still use Acrylic; we wish to continue closely matching the core shell experience. We won't be switching to Mica in that flyout. ([Microsoft doesn't recommend we do this, either.](https://docs.microsoft.com/en-us/windows/apps/design/signature-experiences/materials))
